index.html

```diff
@@ -187,10 +183,42 @@ <h4>About myself</h4>
            external clients to reach their goals, mainly in the educational and 
            financial institutions sector. I can make your projects go ahead, 
            finish on time, improve and grow according to your evolving needs.
-
-        </p>
+          </p>
         <a href="#" class="superButtonR">Download Resume</a>
 
     </section>
+
+    <section id="contactForm">
+
+      <h4>CONTACT US</h4>
+      <p>If you have an application you are interested in developing, a feature that
+        you need built or a project that needs coding, I'd love to help with it.
+      </p>
+      
+      <div class="contact">
+        <form action="https://formspree.io/f/xrgjojpj" method="post">
+          <input type="text" id="name" name="user_name" placeholder="Name" required minlength="1" maxlength="30">
+          <input type="text" id="lastname" name="user_last_name" placeholder="Last name" required minlength="1" maxlength="30">
+          <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+          <textarea id="msg" name="user_message" placeholder="Your message" required minlength="1" maxlength="500"></textarea>
+          
+            <button class="superButton" type="submit">Send</button>
+          
+        </form>
+      </div>
+
+      <hr>
+      <div class="container-WorkSocial">
+        <ul>
+          <li><a href="#" target="new"><i class="fab fa-github social-icon"></i></a></li>
+          <li><a href="#" target="new"><i class="fab fa-linkedin-in social-icon"></i></a></li>
+          <li><a href="#" target="new"><i class="fab fa-facebook social-icon"></i></a></li>
+          <li><a href="#" target="new"><i class="fab fa-twitter social-icon"></i></a></li>
```

Comment:
- [ ] Please reduce the gap between social media icons and make sure it matches the design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166520560-5dd8b9f6-193f-4fc2-8076-8db3239ee898.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166520601-d2afa08a-b561-4dff-b33a-e9cc837ede2a.png)